modules/custom/express_site_metrics/express_site_metrics.module

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/**
+ * @file
+ * Provides statistics reporting for Express web sites.
+ */
+
+$tokenHandlerConstructor = [
+  'metricsUsers' => 'express_site_metrics_users',
+  'secret' => 'express_site_metrics_secret',
+  'dbGet' => 'variable_get',
+  'dbSet' => 'variable_set',
+];
```

Comment:
Is this used?

Reply:
No just missed it when I committed the code. Removing it.